ONSdigital/eq-survey-runner
# Eq 332: Save and Resume - Unique per Collection

### Changes

The user id for save and resume is a hash value consisting of the ru_ref, collection exercise sid and eq_id.
### How to test
1. Enable save and resume; 
   `export EQ_SERVER_SIDE_STORAGE=True`
   `export EQ_SERVER_SIDE_STORAGE_TYPE=Database`
2. Check you can partially complete a survey and then return to it later
3. Check the file /tmp/questionnaire.db and ensure the user id is a hash value.
### Who can test

Anyone apart from @warren-bailey
